assets/pages/monthly-contributions/monthlyContributions.scss

```diff
@@ -106,4 +106,14 @@
 		}
 	}
 
+	// ----- Filler
+
+	.gu-content-filler {
```

Comment:
This piece of CSS is spread across three pages, Is it possible to create a shared component that receives the colour in a prop?